setup.cfg

```diff
@@ -45,6 +45,7 @@ license_files =
    licenses/LICENSE-moment-strftime.txt
    licenses/LICENSE-moment.txt
    licenses/LICENSE-normalize.txt
+   licenses/LICENSES-ui.txt
```

Comment:
This seems unrelated?

Reply:
the pre-commits keep adding that??

Reply:
Hmm that doesn’t make sense, there’s an identical line just below and pre-commit should catch that. Something is probably broken somewhere. Could you temporarily disable pre-commit to remove that line and push to see if CI would let it pass?

Reply:
yep this change should be removed.
this PR should not change `setup.cfg`

Reply:
Chage reverted and pushed.  

My pre-commit hooks also started failing a whole bunch of mypy tests today when I pulled the latest main down, so I may need to unbork something on my environment.  Sorry I missed that.

Reply:
For context, I just now rebased main and ran the pre-commits locally using `pre-commit run --all-files` on it without any of my code changes and it did the same:

```
Update setup.cfg file with all licenses..................................................Failed
- hook id: update-setup-cfg-file
- files were modified by this hook
```